### PR TITLE
Update finding-adding-hosts-dynamically.md

### PR DESCRIPTION
**Pull request by abuehrle**
Took out some wordiness to make the file a little cleaner.  Please take a look to make sure I didn't change the meaning.  

Note: In doing SEO, I've noticed that some of our introductions to the files are a bit curt.  For best SEO results, it is best to mirror the title of the topic in the first sentence, so that it's clear what the topic is.

### CODE REVIEW

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 1]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
 of the existing hosts run:
 
     host# weave connect $NEW_HOST
 
-Other hosts on the Weave network will automatically attempt
+Any other existing hosts on the Weave network will attempt
 to establish connections to the new host as well. 
 
-Alternatively, you can also instruct a peer to forget a 
+###Instructing Peers to Forget a Host
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: It is odd to have this heading but no 'Instructing Peers to Connect to a Host' before that.

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 2]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
 menu_order: 90
 ---
 
-
-To add a host to an existing Weave network, simply launch 
-Weave Net on the host, and then supply the address of at least 
+To add a host to an existing Weave network, launch 
+Weave Net on the host, and supply the address of at least
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: "...at least one existing, running host".

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 3]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
-To add a host to an existing Weave network, simply launch 
-Weave Net on the host, and then supply the address of at least 
+To add a host to an existing Weave network, launch 
+Weave Net on the host, and supply the address of at least 
 one host. Weave Net automatically discovers any other hosts in 
 the network and establishes connections with them if it 
 can (in order to avoid unnecessary multi-hop routing).
 
-In some situations all existing Weave Net hosts may be 
+In some situations existing Weave Net hosts may be
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: The 'all' was important here.

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 4]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
     host# weave connect --replace $NEW_HOST1 $NEW_HOST2
 
-If Weave Net is restarted by Docker it will automatically remember any
-previous connect and forget operations, however if you stop it
-manually and launch it again it will not. If you wish to launch again
-and retain the results of those operations you can do so with
+###Restarting Docker and Weave Net
+
+If Weave Net is restarted by Docker it automatically remembers any
+previous connect and forgets operations, however if you stop it
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: should be 'connect' and 'forget', i.e. quoted.

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 5]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
+manually and launch it again, it will not remember any prior connects.
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: "...any prior 'connect' and 'forget' operations".

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 6]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
-If Weave Net is restarted by Docker it will automatically remember any
-previous connect and forget operations, however if you stop it
-manually and launch it again it will not. If you wish to launch again
-and retain the results of those operations you can do so with
+###Restarting Docker and Weave Net
+
+If Weave Net is restarted by Docker it automatically remembers any
+previous connect and forgets operations, however if you stop it
+manually and launch it again, it will not remember any prior connects. 
+If you want to launch again and retain the results of those operations use
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: s/use/launch Weave Net with. Also needs a comma.

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 7]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
 `--resume`:
 
     host# weave launch --resume
-
-Note that in this case you cannot specify a list of addresses as the
+    
+> **Note:** In this case, you cannot specify a list of addresses, since the
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: s/cannot/must not

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 8]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
 previous peer list is used exclusively.
 
-For complete control over the peer topology, automatic 
-discovery can be disabled using the `--no-discovery` 
-option with `weave launch`. 
+For complete control over the peer topology, disable automatic 
+discovery using the `--no-discovery` option with `weave launch`.
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: "...with the `--no-discovery` option to `weave launch`".

File: site/using-weave/finding-adding-hosts-dynamically.md [hunk 9]
```diff
@@ -3,65 +3,68 @@ title: Adding and Removing Hosts Dynamically
...
+For complete control over the peer topology, disable automatic 
+discovery using the `--no-discovery` option with `weave launch`. 
 
 If discovery if disabled, Weave Net only connects to the 
 addresses specified at launch time and with `weave connect`.
 
-A list of all hosts that a peer has been asked to connect 
-to with `weave launch` and `weave connect` 
-can be obtained using:
+To return a list of all hosts and their peer connections established with
```
Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: s/return/obtain. or 'see'.

File: site/using-weave/finding-adding-hosts-dynamically.md (hunk 8 above)
Comment (abuehrle) [inline site/using-weave/finding-adding-hosts-dynamically.md]: That's a bit of an awkward construction, IMO. That's why I reworded it.

Comment (rade) [inline site/using-weave/finding-adding-hosts-dynamically.md]: I recoil every time I see "use" used.

Comment (abuehrle) [inline site/using-weave/finding-adding-hosts-dynamically.md]: They're tech docs, not poetry!

**Comment by awh** [comment]
@anita @rade what is happening with this folks? It's the only thing blocking #2305.

**Comment by rade** [comment]
@awh #2305 is your PR; it's up to you to decide what goes in it.

**Comment by awh** [comment]
@anita are you happy with this PR as is or did you want to address any more of @rade's comments before I merge it into my branch?

**Comment by abuehrle** [comment]
I think it's fine the way it is @awh

**Pull request merged**